walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -4,24 +4,25 @@
     <meta charset="UTF-8">
     <title>All About Jalyn 🐈</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
-<div class="navigation-bar">
-	<h3><a class="nav" href="/index.html">Home</a></h3>
-    <h3><a  class="nav" href="/resume.html">Resume</a></h3>
-    <h3><a class="nav" href="/socials.html">Socials</a></h3>
+<div id="navigation-bar">
+    <div class="nav"> <span class="nav-btn"> <a class="nav" href="/index.html" style="background-color: white">Home</a> </span> </div>
```

Comment:
nit: Format so no scrolling needed to see entire content. Generally helpful for reviews.

Reply:
No scrolling on my website or no scrolling in github?

Reply:
no scrolling in github, aka, have the content expanded out so I don't have to scroll horizontally to read all the code.